source/develop/release-management/features/network/copy-host-networking.html.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Copy Host Networking
+category: feature
+authors: Ales Musil
+feature_name: Copy Host Networking
+feature_modules: copy-host-networking
+feature_status: In Development
+---
+
+# Copy Host Networking
+
+## Summary
+
+Adding multiple hosts to oVirt via the UI can be rather tedious. The slow point is that the administrator has to go
```

Comment:
```suggestion
Previously, adding hosts to oVirt in the UI was tedious because you, as an administrator, had to set up the host network on each new host.
 
Now, this process is more straightforward because you can set up the network on a single host and copy its network configuration to new hosts. This capability is called Copy Host Networking.
```